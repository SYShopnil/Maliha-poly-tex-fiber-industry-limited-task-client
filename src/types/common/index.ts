export interface ICommonReturnData {
  message: string;
  status: number;
}

export enum EAuth {
  AuthTokenCookieName = "auth",
}

export enum EDataTestId {
  cLoginFormWithSubmit = "cLoginFormWithSubmit",
  cLogoutContainer = "cLogoutContainer",
  SHeaderMain = "SHeaderMain",
  SProfile = "SProfile",
}
