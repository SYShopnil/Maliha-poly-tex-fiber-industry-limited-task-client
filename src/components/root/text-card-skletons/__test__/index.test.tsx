import { render, screen } from "@testing-library/react";
import { EDataTestId } from "@src/types/common";
import { STextCardSkelton } from "..";

describe("When text card skelton  has rendered", () => {
  it("Expect render successfully", () => {
    render(<STextCardSkelton />);
    const myElement = screen.getByRole(EDataTestId.STextCardSkelton);
    expect(myElement).toMatchSnapshot();
  });
});
