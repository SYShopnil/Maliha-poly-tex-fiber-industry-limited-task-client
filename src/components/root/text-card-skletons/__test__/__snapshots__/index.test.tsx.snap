// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When text card skelton  has rendered Expect render successfully 1`] = `
<div
  class="grid grid-cols-12 gap-2"
  role="STextCardSkelton"
>
  <div
    class="col-span-12 md:col-span-6 lg:col-span-4  shadow-lg p-3 "
  >
    <div
      class="text-blue-600 font-bold mb-2 bg-gray-200 h-6 rounded w-1/2 "
    />
    <div
      class="text-gray-700 mb-4 bg-gray-200 h-4 rounded w-3/4"
    />
    <div
      class="flex justify-between"
    >
      <button
        class="bg-gray-200 w-8 text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
      <button
        class="bg-gray-200 w-8  text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
    </div>
  </div>
  <div
    class="col-span-12 md:col-span-6 lg:col-span-4  shadow-lg p-3 "
  >
    <div
      class="text-blue-600 font-bold mb-2 bg-gray-200 h-6 rounded w-1/2 "
    />
    <div
      class="text-gray-700 mb-4 bg-gray-200 h-4 rounded w-3/4"
    />
    <div
      class="flex justify-between"
    >
      <button
        class="bg-gray-200 w-8 text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
      <button
        class="bg-gray-200 w-8  text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
    </div>
  </div>
  <div
    class="col-span-12 md:col-span-6 lg:col-span-4  shadow-lg p-3 "
  >
    <div
      class="text-blue-600 font-bold mb-2 bg-gray-200 h-6 rounded w-1/2 "
    />
    <div
      class="text-gray-700 mb-4 bg-gray-200 h-4 rounded w-3/4"
    />
    <div
      class="flex justify-between"
    >
      <button
        class="bg-gray-200 w-8 text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
      <button
        class="bg-gray-200 w-8  text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
    </div>
  </div>
  <div
    class="col-span-12 md:col-span-6 lg:col-span-4  shadow-lg p-3 "
  >
    <div
      class="text-blue-600 font-bold mb-2 bg-gray-200 h-6 rounded w-1/2 "
    />
    <div
      class="text-gray-700 mb-4 bg-gray-200 h-4 rounded w-3/4"
    />
    <div
      class="flex justify-between"
    >
      <button
        class="bg-gray-200 w-8 text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
      <button
        class="bg-gray-200 w-8  text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
    </div>
  </div>
  <div
    class="col-span-12 md:col-span-6 lg:col-span-4  shadow-lg p-3 "
  >
    <div
      class="text-blue-600 font-bold mb-2 bg-gray-200 h-6 rounded w-1/2 "
    />
    <div
      class="text-gray-700 mb-4 bg-gray-200 h-4 rounded w-3/4"
    />
    <div
      class="flex justify-between"
    >
      <button
        class="bg-gray-200 w-8 text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
      <button
        class="bg-gray-200 w-8  text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
    </div>
  </div>
  <div
    class="col-span-12 md:col-span-6 lg:col-span-4  shadow-lg p-3 "
  >
    <div
      class="text-blue-600 font-bold mb-2 bg-gray-200 h-6 rounded w-1/2 "
    />
    <div
      class="text-gray-700 mb-4 bg-gray-200 h-4 rounded w-3/4"
    />
    <div
      class="flex justify-between"
    >
      <button
        class="bg-gray-200 w-8 text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
      <button
        class="bg-gray-200 w-8  text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
    </div>
  </div>
  <div
    class="col-span-12 md:col-span-6 lg:col-span-4  shadow-lg p-3 "
  >
    <div
      class="text-blue-600 font-bold mb-2 bg-gray-200 h-6 rounded w-1/2 "
    />
    <div
      class="text-gray-700 mb-4 bg-gray-200 h-4 rounded w-3/4"
    />
    <div
      class="flex justify-between"
    >
      <button
        class="bg-gray-200 w-8 text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
      <button
        class="bg-gray-200 w-8  text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
    </div>
  </div>
  <div
    class="col-span-12 md:col-span-6 lg:col-span-4  shadow-lg p-3 "
  >
    <div
      class="text-blue-600 font-bold mb-2 bg-gray-200 h-6 rounded w-1/2 "
    />
    <div
      class="text-gray-700 mb-4 bg-gray-200 h-4 rounded w-3/4"
    />
    <div
      class="flex justify-between"
    >
      <button
        class="bg-gray-200 w-8 text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
      <button
        class="bg-gray-200 w-8  text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
    </div>
  </div>
  <div
    class="col-span-12 md:col-span-6 lg:col-span-4  shadow-lg p-3 "
  >
    <div
      class="text-blue-600 font-bold mb-2 bg-gray-200 h-6 rounded w-1/2 "
    />
    <div
      class="text-gray-700 mb-4 bg-gray-200 h-4 rounded w-3/4"
    />
    <div
      class="flex justify-between"
    >
      <button
        class="bg-gray-200 w-8 text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
      <button
        class="bg-gray-200 w-8  text-gray-600 py-2 px-4 rounded-md cursor-not-allowed"
      />
    </div>
  </div>
</div>
`;
